rotkehlchen/api/v1/schemas.py

```diff
@@ -1785,7 +1791,13 @@ def make_assets_post_query(
     ) -> Dict[str, Any]:
         ignored_assets_identifiers = None
         with self.db.user_write() as write_cursor, GlobalDBHandler().conn.read_ctx() as globaldb_read_cursor:  # noqa: E501
-            if data['include_ignored_assets'] is False:
+            ignored_assets_filter: Optional[Literal['only_ignored', 'exclude_ignored']] = None
+            if data['only_ingored_assets'] is True:
+                ignored_assets_filter = 'only_ignored'
```

Comment:
here use the operator directly as the value. Also can make it shorter with the ternary operator

Reply:
I made it like this for better readability

Reply:
It's not. Please do what I asked